Formula/trailscraper.rb

```diff
@@ -0,0 +1,102 @@
+class Trailscraper < Formula
+  include Language::Python::Virtualenv
+
+  desc "Tool to get valuable information out of AWS CloudTrail"
+  homepage "https://github.com/flosell/trailscraper"
+  url "https://github.com/flosell/trailscraper/archive/0.5.1.tar.gz"
+  sha256 "6793353bef9cf17379d556e7811d225ee852c6c2e975deb7219d684be7b12c5b"
+
+  depends_on "python"
```

Comment:
Maybe you can use python 3.8?
```suggestion
  depends_on "python@3.8"
```

Reply:
Tested and updated